tests/components/openevse/test_sensor.py

```diff
@@ -0,0 +1,75 @@
+"""Tests for the OpenEVSE sensor platform."""
```

Comment:
We're missing tests of the config flow. It requires 100% coverage.

Reply:
Done I believe.